dist/style.css

```diff
@@ -0,0 +1,62 @@
+#gall_root {
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+  max-width: 50%;
+}
+
+#gall_wrapper {
+  flex-direction: column;
+}
+
+#gall_main {
+  width: 100%;
+  max-width: 576px;
+  max-height: 576px;
+}
+
+#gall_main img {
+  max-width: 100%;
+  height: auto;
+}
+
+#gall_caption {
+  font-family: Arial, sans-serif;
```

Comment:
consider moving your font declaration to the highest level container as amazon uses only one font.